staticmemory/memory-bucket-optimizer/generate_test_logs.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+#
+# Copyright (C) 2025 wolfSSL Inc.
+#
+# This file is part of wolfSSL.
+#
+# wolfSSL is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# wolfSSL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+
+
+# Script to generate test memory logs and run the optimizer
+
+# Set up directories
+WOLFSSL_DIR=~/repos/wolfssl
```

Comment:
Used a passed in argument here. For example:

```
if [ -z $1 ]; then                                                              
    echo Expecting absolute path to root wolfssl directory                      
    exit 1                                                                      
else                                                                            
    WOLFSSL_DIR="$1"                                                            
fi  
```

Do this for the other  scripts too that are expecting a WOLFSSL_DIR set.